app/components/overview-info-form/template.hbs

```diff
@@ -17,6 +12,6 @@
       {{/if}}
     {{/bs-form-group}}
   {{/each-in}}
-  <div class='btn btn-primary pull-right {{if (v-get this 'isInvalid') "disabled" "enabled"}}' {{action 'nextSection'}}>Continue</div>
+  <div class='btn btn-primary pull-right {{if (v-get this 'isInvalid') "disabled" "enabled"}}' {{action 'nextSection'}}>{{t 'global.continueLabel'}}</div>
```

Comment:
I'm not sure "enabled" is a valid bootstrap class. The third argument to the if helper is optional, so you could just omit it.